source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -682,6 +682,11 @@ void buildIntermediateResultForText(ACRView *rootView, ACOHostConfig *hostConfig
     auto markdownString = markDownParser->TransformToHtml();
     NSString *parsedString = (markDownParser->HasHtmlTags()) ? [NSString stringWithCString:markdownString.c_str() encoding:NSUTF8StringEncoding] : [NSString stringWithCString:markDownParser->GetRawText().c_str() encoding:NSUTF8StringEncoding];
 
+    if (markDownParser->HasHtmlTags() && ([parsedString containsString:@"\n"] || [parsedString containsString:@"\r"])) {
```

Comment:
you can do something like this.
    parsedString = [parsedString stringByReplacingOccurrencesOfString:@"[\\n\\r]" withString:@"<br>" options:NSRegularExpressionSearch range:NSMakeRange(0, [parsedString length])];

Reply:
good point, thanks!